src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.Common.cs

```diff
@@ -163,122 +163,165 @@ public void SeekSkipsEmptySegments()
         }
 
         [Fact]
-        public void SeekEmptySkipDoesNotCrossPastEnd()
+        public void TryGetReturnsEmptySegments()
```

Comment:
It's difficult to review (in the diff) what tests are changing here. Are we modifying any existing tests here or just adding some (and git diff is getting confused)?

Reply:
We changed the behavior so old tests are invalid and replaced with new ones that show the new behavior.

Reply:
I see. We are only modifying the tests involving TryGet here, correct?

Are test cases like these no longer valid?
```C#
        [Fact]
        public void TryGetSkipDoesNotCrossPastEndUsingGetPositionWithOffsetAndOrigin()
        {
            var bufferSegment1 = new BufferSegment<byte>(new byte[100]);
            BufferSegment<byte> bufferSegment2 = bufferSegment1.Append(new byte[0]);
            BufferSegment<byte> bufferSegment3 = bufferSegment2.Append(new byte[0]);
            BufferSegment<byte> bufferSegment4 = bufferSegment3.Append(new byte[100]);

            var buffer = new ReadOnlySequence<byte>(bufferSegment1, 0, bufferSegment2, 0);
            SequencePosition c1 = buffer.GetPosition(0, buffer.Start);
            Assert.Equal(0, c1.GetInteger());
            Assert.Equal(bufferSegment1, c1.GetObject());

            ReadOnlyMemory<byte> data;
            Assert.True(buffer.TryGet(ref c1, out data, advance: true));

            Assert.Equal(0, c1.GetInteger());
            Assert.Equal(null, c1.GetObject());

            Assert.False(buffer.TryGet(ref c1, out data, advance: true));
        }
```

Reply:
No, they were checking that TryGet skips empty segments which it doesn't do anymore

Reply:
Except the tests were testing `GetPosition` which uses `Seek` not `TryGet` and you've not changed `Seek`, so  `Seek`/`GetPosition` now has less tests

Reply:
K, I'll bring them back

Reply:
You just swapped where the test was :)

Reply:
Is ok, you just swapped where the test was :) 

(According to GitHub's bad diffing)

Reply:
Removed tests were only doing `GetPosition(0)`

Reply:
Was specifically concerned about `SeekEmptySkipDoesNotCrossPastEnd`; but its still there, just "moved"

Reply:
Pavel, how has code coverage been affected by these test changes?

Reply:
Didn't change at all, these are commonly used code paths.